AirCasting/Graph/Graph.swift

```diff
@@ -73,13 +73,13 @@ class UI_PollutionGraph: UIView {
         lineChartView.leftYAxisRenderer = renderer
     }
     
-    func updateWith(values: [Float]) {
+    func updateWith(thresholdValues: [Float]) {
         guard let renderer = renderer else { return }
-        renderer.thresholds = values
+        renderer.thresholds = thresholdValues
         
         //min & max yaxis values
-        lineChartView.leftAxis.axisMinimum = Double(values.first ?? 0)
-        lineChartView.leftAxis.axisMaximum = Double(values.last ?? 200)
+        lineChartView.leftAxis.axisMinimum = Double(thresholdValues.first ?? 0)
+        lineChartView.leftAxis.axisMaximum = Double(thresholdValues.last ?? 200)
         print("\(lineChartView.leftAxis.axisMinimum) - \(lineChartView.leftAxis.axisMaximum)")
         
         // Needed to redraw the chart
```

Comment:
Is the code under not understandable? are the comments needed? the code seams pretty self explanatory. Seams more like for a tutorial land not production code

Reply:
Well, tbh I was adding those comments for myself. Creating the graph was a tough task for me, so I was describing lots of code to make the task easier. 
I'm removing this comment.